src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
Don’t use static methods in your solution.

Reply:
Useful link: https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5